07.ls_object/lib/display.rb

```diff
@@ -30,20 +30,17 @@ def initialize(paths)
   end
 
   def render(long: false)
-    if long
-      render_long_list
-    else
-      render_short_list
-    end
+    long ? render_long_list : render_short_list
   end
 
   private
 
   def render_long_list
     max_widths = find_max_widths
     total = "total #{@file_statuses.sum(&:blocks)}"
-    long_list = @file_statuses.map { |status| format_file_status(status, max_widths) }
-    [total, long_list]
+    long_list = @file_statuses.map { |status| format_file_status(status, max_widths) }.join("\n")
+    # [total, long_list]
+    "#{total}\n#{long_list}"
```

Comment:
```suggestion
    details = @file_statuses.map { |status| format_file_status(status, max_widths) }
    [total, *details].join("\n")
```

みたいにするとちょっとスマートかも。

あと、コメントの消し忘れに注意！
https://blog.jnito.com/entry/2021/01/30/172607

Reply:
ありがとうございました！
はやる気持ちから、しょうもないコメント削除ミスをしてしまいました😂
いただいた点はすべて修正しました。
メンターさんや他の受講生の方のコードも見てみます！